kedro/extras/datasets/pandas/gbq_dataset.py

```diff
@@ -137,12 +137,18 @@ def _describe(self) -> Dict[str, Any]:
 
     def _load(self) -> pd.DataFrame:
         sql = "select * from {}.{}".format(self._dataset, self._table_name)  # nosec
-        return pd.read_gbq(
-            sql,
-            project_id=self._project_id,
-            credentials=self._credentials,
-            **self._load_args
-        )
+        if "query" not in self._load_args:
+            self._load_args.update({"query": sql})
+
+        self._load_args["query"] = str(self._load_args["query"])
+        try:
+            return pd.read_gbq(
+                project_id=self._project_id,
+                credentials=self._credentials,
+                **self._load_args
+            )
+        except ValueError:
+            raise ValueError()
```

Comment:
why is this needed? can you not skip the `try/except`? We don't do this for any of our other datasets afaics